tests/lib/rules/no-todo.js

```diff
@@ -0,0 +1,77 @@
+/**
+ * @fileoverview Forbid the use of QUnit.todo
+ * @author Baptiste Doucerain
+ */
+"use strict";
+
+//------------------------------------------------------------------------------
+// Requirements
+//------------------------------------------------------------------------------
+
+const rule = require("../../../lib/rules/no-todo"),
+    RuleTester = require("eslint").RuleTester;
+
+//------------------------------------------------------------------------------
+// Tests
+//------------------------------------------------------------------------------
+
+const ruleTester = new RuleTester();
+
+ruleTester.run("no-todo", rule, {
+    valid: [
+        "QUnit.module.test('Name', function() { });",
+        "QUnit.test('Name', function() { });",
+        "module.test('Name', function() { });",
+        "test('Name', function() { });",
+        "foo.todo('Name', function() { });",
+        "foo.bar.todo('Name', function() { });",
+        "module.foo.todo('Name', function() { });",
+        "test.foo.todo('Name', function() { });",
+        "todo.foo('Name', function() { });"
+    ],
+
+    invalid: [
+        {
+            code: "QUnit.module.todo('Name', function() { });",
+            errors: [{
+                messageId: "noQUnitTodo",
+                data: { callee: "QUnit.module.todo" }
+            }]
+        },
+        {
+            code: "QUnit.todo('Name', function() { });",
+            errors: [{
+                messageId: "noQUnitTodo",
+                data: { callee: "QUnit.todo" }
+            }]
+        },
+        {
+            code: "QUnit.test.todo('Name', function() { });",
+            errors: [{
+                messageId: "noQUnitTodo",
+                data: { callee: "QUnit.test.todo" }
+            }]
+        },
+        {
+            code: "module.todo('Name', function() { });",
+            errors: [{
+                messageId: "noQUnitTodo",
+                data: { callee: "module.todo" }
+            }]
+        },
+        {
+            code: "todo('Name', function() { });",
```

Comment:
If I understand correctly, you are saying that `todo` is not a valid way to call the method. It will be either used like this: `QUnit.test.todo` or `Qunit.todo`.

However in our app we are using the `todo` like this :
```
import { todo } from 'qunit';
...
module( 'foo', function () {
    todo('bar', function () {
        ...
    }
}
```

So it would be very interesting for us if the `todo()` method can be considered as invalid/warning.

I can't find any example of this usage in the API documentation  😣

Reply:
@platinumazure sorry for the ping, any thoughts on this subject ?

Reply:
@DBattou Sorry for missing your comment, I've been pretty busy this week. I'll have a more complete response when I have a chance to review, but long story short, I hadn't thought of the named import case and you probably don't need to change anything in this PR.

We eventually want to distinguish between true global `todo` and named import `todo`. However, that's a bit of work and might require redesigning not only the rule, but the utilities classes. So I'm not going to require it in this PR as the likelihood of a false positive should be pretty low (and users can use ESLint disable comments if they need to).

Thanks for your continued patience on this!

Reply:
Okay, so what I'd like to see here is for us to change this test to include the import from QUnit (`import { todo } from "qunit"`). That way, if we can fix the underlying logic, we don't have to change the test (it would still work since it isn't a global variable in this test case).

To put it another way, the test shows our intent to support the named import case, and does not suggest that we want to support a global variable `todo` here (even though the implementation currently does).

Make sense?